obspy/io/seiscomp/tests/test_event.py

```diff
@@ -79,7 +84,13 @@ def cmp_write_xslt_file(self, quakeml_file, sc3ml_file, validate=True,
             if validate:
                 self.assertTrue(validate_sc3ml(tf.name))
             filepath_cmp = os.path.join(self.path, sc3ml_file)
-            self.assertTrue(filecmp.cmp(filepath_cmp, tf.name))
+            tf.seek(0)
+            dat1 = tf.read().decode("utf-8").splitlines()
+            with open(filepath_cmp, 'rb') as f:
+                dat2 = f.read().decode("utf-8").splitlines()
+            self.assertEqual(len(dat1), len(dat2))
+            for d1, d2 in zip(dat1, dat2):
+                self.assertEqual(d1, d2)
```

Comment:
Can you wrap this in a short helper method as its repeated a couple of times?

Reply:
no :)